frontend/src/store/index.js

```diff
@@ -475,19 +509,48 @@ const getters = {
     return getters['shoots/initialNewShootResource']
   },
   hasGardenTerminalAccess (state, getters) {
-    return getters.isTerminalEnabled && getters.isAdmin
+    return getters.canUseTerminalFeature && getters.isAdmin
```

Comment:
why isAdmin?

Reply:
garden terminal currently is only available for the admin